addons/pinniped/post-deploy/pkg/configure/helper_test.go

```diff
@@ -0,0 +1,246 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package configure
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"sync"
+	"testing"
+
+	"github.com/vmware-tanzu/tanzu-framework/addons/pinniped/post-deploy/pkg/configure/supervisor"
+
+	kubeerrors "k8s.io/apimachinery/pkg/api/errors"
+
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	kubefake "k8s.io/client-go/kubernetes/fake"
+	kubetesting "k8s.io/client-go/testing"
+)
+
+// nolint:funlen
+func TestCreateOrUpdatePinnipedInfo(t *testing.T) {
+	const namespace = "kube-public"
+	const name = "pinniped-info"
+	supervisorPinnipedInfo := supervisor.PinnipedInfo{
```

Comment:
might this variable be more appropriately named `managementClusterPinnipedInfo`? i feel like when i hear "supervisor" i think "pinniped", but when i hear "managementCluster" i think "tkg". i think the pinniped-info is tied more closely to the type of _tkg cluster_ rather than _pinniped cluster_, given that the concierge runs on every cluster (management and workload).